datasets/kz/kz_companies/kz_companies.yml

```diff
@@ -0,0 +1,28 @@
+name: kz_companies
+title: State Register of legal entities in the Republic of Kazakhstan
+prefix: kz
+disabled: false
+entry_point: crawler
+coverage:
+  frequency: weekly
+  start: 2023-03-21
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+deploy:
+  # premium: true
+resolve: false
+ci_test: false
+http:
+  total_retries: 5
+  backoff_factor: 3
+  retry_statuses: [503, 429, 403]
+
+summary: |
+  All companies in Kazakhstan, with information on their directors.
```

Comment:
What could be good is to add a description field which conveys the breadth of different types of entities in the registry, if what you see the data really matches what we see in the state registration of entities section on https://egov.kz/cms/en/services/pass042com_mu